firestore.rules

```diff
@@ -53,9 +53,13 @@ service cloud.firestore {
         return get(/databases/$(database)/documents/worlds/$(venueData().worldId)).data;
       }
 
+      function checkIfSpaceOrWorldOwner(userId) {
+        return $(userId) in role('admin').users && ($(userId) in venueData().owners || $(userId) in world().owners)
```

Comment:
```suggestion
        return $(userId) in role('admin').users || $(userId) in venueData().owners || $(userId) in world().owners
```
My understanding of the "admin" role is that is for Sparkle staff to be able to edit anything.

This is a nice solution btw. Glad it worked out nice and tidy :)